modules/nw-configuring-lb-allowed-source-ranges.adoc

```diff
@@ -0,0 +1,30 @@
+// Modules included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-load-balancer-allowed-source-ranges.adoc
+
+:_content-type: PROCEDURE
+[id="nw-configuring-lb-allowed-source-ranges_{context}"]
+= Configuring Load Balancer allowed source ranges
+
+By configuring Load Balancer allowed source ranges, you can limit the access to the Load Balancer to a specified list of IP address ranges on the cloud provider for the ingress. The Ingress Operator reconciles the Load Balancer Service and sets the `.Spec.LoadBalancerSourceRanges` based on `AllowedSourceRanges`.
```

Comment:
```suggestion
By configuring Load Balancer allowed source ranges, you can limit the access to the Load Balancer for the `ingresscontroller` to a specified list of IP address ranges. The Ingress Operator reconciles the Load Balancer Service and sets the `.Spec.LoadBalancerSourceRanges` based on `AllowedSourceRanges`.
```
I think it is a bit more clear this way.

Reply:
Addressed